### ISSUE TITLE
Related content promos should only output as a list if more than one promo

### ISSUE THREAD

**Issue by greenc05** [BBC-archive/psammead#3295]
**Describe the bug**
A single promo in the related content component is contained within a HTML list as a list item, promos should only be in a list with list items if there is more than one promo.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/somali/war-50722888
2. Inspect the code for the related content promos 
3. You will see list and list item HTML elements even though there is only 1 item/promo

**Expected behaviour**
If there is only one promo then a list with list item should not be output. There should only be a list with list items if we have a list e.g. more than one promo

**Screenshots**
<img width="771" alt="Screen Shot 2020-03-24 at 11 43 32" src="https://user-images.githubusercontent.com/1061401/77422025-ba030880-6dc4-11ea-9f32-110e0d842ad4.png">

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Comment by mykeels** [comment]
I think this should be in Simorgh. I don't think the fix for this will come from the psammead components. 

Also, see https://github.com/bbc/simorgh/blob/latest/src/app/containers/FrontPageSection/index.jsx#L124-L134 for an example of a fix.

**Comment by mykeels** [comment]
Now created [here](https://github.com/bbc/simorgh/issues/6047), and assigned to myself. I've marked this as a duplicate.

https://github.com/bbc/simorgh/issues/6047
